server/multitenancy/tenant_resolver.ts

```diff
@@ -75,10 +75,10 @@ export function resolveTenant(
  */
 export function isMultitenantPath(request: KibanaRequest): boolean {
   return (
-    request.url.path?.startsWith('/elasticsearch') ||
-    request.url.path?.startsWith('/api') ||
-    request.url.path?.startsWith('/app') ||
-    request.url.path === '/'
+    request.url.pathname?.startsWith('/elasticsearch') ||
```

Comment:
Curious why this is needed as I cannot find another file which has this change.

Reply:
the `path` includes query string , while the `pathname` only includes the URL path portion. this is a fix of misuse of fields